src/ripple/app/misc/FeeVoteImpl.cpp

```diff
@@ -28,14 +28,13 @@ namespace ripple {
 
 namespace detail {
 
-template <typename Integer>
 class VotableInteger
 {
 private:
-    using map_type = std::map <Integer, int>;
+    using Integer = XRPAmount;
```

Comment:
Is there a particular reason you stuck with the `Integer` type alias? Now that the class is no longer generic, it seems like it's specifically a class about voting on `XRPAmounts`.

Reply:
Mainly laziness. But it's back to being more generic again, so that's awesome.